app/api/pokemon.js

```diff
@@ -104,13 +152,13 @@ module.exports = {
      * @apiUse NoRecords
      * 
      */
-    getIconById: function(req, res) {
+    getSvgIconById: function(req, res) {
         logger.info('Get Pokemon icon of particular pokemon by id');
 
         pokemon.getIconById(req.params.id, function(success, message) {
             if(success === 1) {
-                res.writeHead(200, {'Content-Type': message[0].icon.contentType });
-                res.end(message[0].icon.data, 'binary');
+                res.writeHead(200, {'Content-Type': message[0].iconSvg.contentType });
+                res.end(message[0].iconSvg.data, 'binary');
```

Comment:
@samitsv - Add these API names to apidoc.json. Otherwise, they will not show up in the API documentation